docs/client_advanced.rst

```diff
@@ -472,8 +472,8 @@ Custom socket creation
 ^^^^^^^^^^^^^^^^^^^^^^
 
 If the default socket is insufficient for your use case, pass an optional
-`socket_factory` to the :class:`~aiohttp.TCPConnector`, which implements
-`SocketFactoryType`. This will be used to create all sockets for the
+``socket_factory`` to the :class:`~aiohttp.TCPConnector`, which implements
+``SocketFactoryType``. This will be used to create all sockets for the
```

Comment:
You should be able to link to it now that you have it as a class in https://github.com/aio-libs/aiohttp/pull/10568/files#diff-89957958d0a4f5bec39bd0390ff523269ce26597a7c20abdc25818ce658c126bR1137